app/services/foreman_rh_cloud/cloud_ping_service.rb

```diff
@@ -2,28 +2,6 @@
 
 module ForemanRhCloud
   class CloudPingService
-    class TokenPing
-      include ForemanRhCloud::CloudAuth
-
-      attr_accessor :logger
-
-      def initialize(logger)
-        @logger = logger
-      end
-
-      def ping
```

Comment:
We should probably also drop the part in the UI that displays API token status:
![Screenshot (88)](https://user-images.githubusercontent.com/26363699/168474480-e31fb65c-d6e0-48b7-bc4a-1e0515f77856.png)